.gitignore

```diff
@@ -2,6 +2,7 @@
 **/build-sim/
 *.b#*
 *.bin
+.cache/
```

Comment:
This seems like it was added by accident, right?

Reply:
Reverted that change :+1: